src/fake-timers-src.js

```diff
@@ -109,6 +109,7 @@ const globalObject = require("@sinonjs/commons").global;
  * @property {number} createdAt
  * @property {boolean} immediate
  * @property {number} id
+ * @property {Error} [error]
```

Comment:
I _think_ this is how you say `optional`?

Reply:
I think so